pandas/sparse/array.py

```diff
@@ -460,30 +471,38 @@ def sum(self, axis=None, dtype=None, out=None):
             nsparse = self.sp_index.ngaps
             return sp_sum + self.fill_value * nsparse
 
-    def cumsum(self, axis=0, dtype=None, out=None):
+    def cumsum(self, axis=0, *args, **kwargs):
         """
         Cumulative sum of values. Preserves locations of NaN values
 
-        Extra parameters are to preserve ndarray interface.
+        Note that the `axis` parameter is not used in the implementation
+        but is placed in the signature to maintain a more uniform interface.
 
         Returns
         -------
-        cumsum : Series
+        cumsum : SparseArray if `self` has a null `fill_value` and a
```

Comment:
If you check it yourself, you'll see that the initial documentation was inaccurate.